packages/server/lib/controllers/sync.controller.ts

```diff
@@ -401,26 +407,20 @@ class SyncController {
 
             if (isOk(actionResponse)) {
                 res.send(actionResponse.res);
+
                 return;
             } else {
+                span.setTag('error', actionResponse.err);
                 errorManager.errResFromNangoErr(res, actionResponse.err);
+
                 return;
             }
-        } catch (e: any) {
-            const content = e.message || 'Unknown error';
-
-            await metricsManager.capture(
-                MetricTypes.ACTION_FAILURE,
-                `Action: ${action_name} failed with error: ${content}`,
-                LogActionEnum.ACTION,
-                {
-                    input: JSON.stringify(input, null, 2),
-                    actionName: action_name
-                },
-                `actionName:${action_name}`
-            );
```

Comment:
I still believe it's a mistake to rely on Span for metrics, we can still do both if we want to cover debugging and metrics.

Reply:
Can you expand on why you think it is a mistake to use Span for metrics?

Reply:
re: it's logs not metrics 👌🏻